### PR TITLE
DOC: Update jnp.min/amin docs

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
Part of #21461.

### CODE REVIEW

File: jax/_src/numpy/reductions.py [hunk 1]
```diff
@@ -409,10 +409,79 @@ def _reduce_min(a: ArrayLike, axis: Axis = None, out: None = None,
...
         keepdims: bool = False, initial: ArrayLike | None = None,
         where: ArrayLike | None = None) -> Array:
+  r"""Return the minimum of array elements along a given axis.
+
+  JAX implementation of :func:`numpy.min`.
+
+  Args:
+    a: Input array.
+    axis: Axis along which the minimum to be computed. If axis=None, the minimum
+      is computed along both the axes. Default=None.
```
Comment (jakevdp) [inline jax/_src/numpy/reductions.py:423]: Delete lines 422-423?

File: jax/_src/numpy/reductions.py [hunk 2]
```diff
@@ -409,10 +409,79 @@ def _reduce_min(a: ArrayLike, axis: Axis = None, out: None = None,
...
+
+  See also:
+    - :func:`jax.numpy.max`: Compute the maximum of array elements along a given
+      axis.
+    - :func:`jax.numpy.sum`: Compute the sum of array elements along a given axis.
+    - :func:`jax.numpy.prod`: Compute the product of array elements along a given
+      axis.
+
+  Examples:
+
```
Comment (jakevdp) [inline jax/_src/numpy/reductions.py:445]: Remove blank line (this will cause a lint error once #22037 is merged)

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: jax/_src/numpy/reductions.py (hunk 1 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/reductions.py]: Deleted.

File: jax/_src/numpy/reductions.py (hunk 2 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/reductions.py]: Thanks! Removed the blank line and squashed into single commit.

**Review by jakevdp — APPROVED** [review]
Thanks!

**Pull request merged**